### ISSUE TITLE
Brave interferes with bang processing

### ISSUE THREAD

**Issue by lslenders** [psidex/CustomBangSearch#48]
Hi, I  will illustrate this bug with an example: 

`!wf`, is configured to 'https://workflowy.com/%s' and has been working fine until recently. 
now typing  in `!wf`, redirects to 'https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Accueil_principal'

I suspect this is bc the duckduckgo bang was updated. Removing the existing bang and adding it again made it work like a charm again. Now `!wf` redirects to my initial `https://workflowy.com/%s`

FYI

**Comment by lslenders** [comment]
quickest fix / parking is adding  a line about this in the readme ;)

**Comment by psidex** [comment]
I'm not sure why or how this would happen... What is your default search engine?

**Comment by lslenders** [comment]
I'm using Brave :
Brave 1.71.121, Chromium 130.0.6723.91

**Comment by psidex** [comment]
Thanks, what search engine does your address bar default to? is it DDG?

**Comment by lslenders** [comment]
.. it says Brave in the settings...

<img width="891" alt="Screenshot 2024-11-07 at 14 01 04" src="https://github.com/user-attachments/assets/c3cfcebb-c6a8-4768-b755-f93e29f3cba2">

**Comment by psidex** [comment]
Apologies, I forget that Brave is a search engine as well as a browser!

![image](https://github.com/user-attachments/assets/0edd2477-695d-498a-a1c4-3ce4bd2d3767)

Looks like it was Brave catching the bang and using it's own system before CBS could intercept. I'm not sure why updating it in your settings made it work again. I will have to investigate, thanks

**Comment by lslenders** [comment]
Yes you are right! - I did some more experimenting, and here is the catch: when I update the bang, it will only last for that browser session! After a complete reboot of the machine it will redirect to (in this example) the French Wiki (instead of Workflowy, the link I want it to redirect).

**Comment by psidex** [comment]
As there hasn't been any discussion on this for a couple of years I'm going to close this issue
